apps/kg/dataloader/KGDataset.py

```diff
@@ -66,9 +66,11 @@ def __init__(self, path, name):
         self.n_entities = len(self.entity2id)
         self.n_relations = len(self.relation2id)
 
-        self.train = self.read_triple(path, 'train')
-        self.valid = self.read_triple(path, 'valid')
-        self.test = self.read_triple(path, 'test')
+        if read_triple == True:
```

Comment:
why do you want to turn off reading triples?

Reply:
For kvserver, we only need to know the data shape instead of the whole data triples.